src/test/java/games/strategy/engine/framework/ArgParserTest.java

```diff
@@ -38,37 +39,24 @@ public void argsTurnIntoSystemProps() {
 
   @Test
   public void emptySystemPropertiesCanBeSet() {
-    ArgParser.handleCommandLineArgs(new String[] {"a="}, new String[] {"a"});
+    new ArgParser(Collections.singleton("a")).handleCommandLineArgs(new String[] {"-Pa="});
     assertThat("expecting the system property to be empty string instead of null",
         System.getProperty("a"), is(""));
   }
 
   @Test
-  public void malformedInputThrowsException() {
```

Comment:
I had to drop this test, because the cli parser is very forgiving if an argument is not a flag